model-engine/model_engine_server/common/constants.py

```diff
@@ -0,0 +1,10 @@
+from pathlib import Path
+
+BILLING_POST_INFERENCE_HOOK: str = "billing"
+CALLBACK_POST_INFERENCE_HOOK: str = "callback"
+READYZ_FPATH: str = "/tmp/readyz"
+DEFAULT_CELERY_TASK_NAME: str = "hosted_model_inference.inference.async_inference.tasks.predict"
+LIRA_CELERY_TASK_NAME: str = "ml_serve.celery_service.exec_func"
```

Comment:
are these two still used?

Reply:
Yeah there are still legacy endpoints on the default task name